website/content/docs/connect/gateways/api-gateway/configuration/gatewaypolicy.mdx

```diff
@@ -0,0 +1,259 @@
+---
+
+# GatewayPolicy
+
+This topic provides reference information for the API gateway policy resource. These resources define policies for listeners on API gateways deployed to Kubernetes-orchestrated networks.
+
+## Configuration model
+
+The following list outlines field hierarchy, data types, and requirements in a gateway policy resource. Click on a property name to view additional details, including default values.
+
+- [`apiVersion`](#apiversion): string | required | must be set to `consul.hashicorp.com/v1alpha1`
+- [`kind`](#kind): string | required | must be set to `GatewayPolicy`
+- [`metadata`](#metadata): map | required
+   - [`name`](#metadata-name): string | required 
+   - [`namespace`](#metadata-namespace): string  | `default`
+- [`spec`](#spec): map | required
+   - [`targetRef`](#spec-targetref): map | required
+      - [`namespace`](#spec-targetref): string | `default` 
+      - [`name`](#spec-targetref): string | required 
+      - [`kind`](#spec-targetref): string | required | must be set to `Gateway`
+      - [`group`](#spec-targetref): string | required 
+      - [`sectionName`](#spec-targetref): string 
+   - [`override`](#spec-override): map | required 
+      - [`jwt`](#spec-override-jwt): map | required 
+         - [`providers`](#spec-override-providers): list | required 
+            - [`name`](#spec-override-providers): string | required 
+            - [`verifyClaims`](#spec-override-providers): map | required 
+               - [`path`](#spec-override-providers): list of strings | required 
+               - [`value`](#spec-override-providers): string | required 
+   - [`default`](#spec-default): map | required 
+      - [`jwt`](#spec-default-jwt): map | required 
+         - [`providers`](#spec-default-providers): list | required 
+            - [`name`](#spec-default-providers): string | required 
+            - [`verifyClaims`](#spec-default-providers): map | required 
+               - [`path`](#spec-default-providers): list of strings | required 
+               - [`value`](#spec-default-providers): string | required 
+
+## Complete configuration
+
+When every field is defined, a gateway policy has the following form: 
+
+```yaml
+apiVersion: consul.hashicorp.com/v1alpha1
+kind: GatewayPolicy
+metadata:
+   name: <name for the policy>
+   namespace: <Consul namespace the policy applies to>
+spec:
+  targetRef:
+    name: gateway
+    kind: Gateway
+    group: gateway.networking.kuberenetes.io
+    sectionName: <name of a specific listener the policy applies>
+  override:
+    jwt:
+      providers:
+      - name: "<name of the provider all listeners verify against>"
+        verifyClaims:
+        - path:
+          - "aud"
+          value: "api.apps.organization.com"
+  default:
+    jwt:
+      providers:
+      - name: "<name of the default provider>"
```

Comment:
so this provider can be the same as the one in the `override` it's more that these just need to reference by name the specific JWT provider that they should be verified against